censys/cli/commands/asm.py

```diff
@@ -530,6 +531,74 @@ def cli_delete_saved_query_by_id(args: argparse.Namespace):
         sys.exit(1)
 
 
+def cli_execute_saved_query_by_name(args: argparse.Namespace):
+    """Execute saved query by name subcommand.
+
+    Args:
+        args (Namespace): Argparse Namespace.
+    """
+    s = InventorySearch(args.api_key)
+    q = SavedQueries(args.api_key)
+    # get query from name
+    queries = q.get_saved_queries(args.query_name, 1, 1)
+    if not queries["results"]:
+        console.print("No saved query found with that name.")
+        sys.exit(1)
+    query = queries["results"][0]["query"]
+
+    try:
+        res = s.search(None, query, args.page_size, None, args.sort, args.fields)
+        console.print_json(json.dumps(res))
```

Comment:
Does this need the json.dumps? I think it may do that automatically.

Reply:
`print_json` takes in a `str`, where `res` is a `dict`